composer.json

```diff
@@ -22,7 +22,7 @@
     "require": {
         "php": "~5.6|~7.0",
         "emojione/emojione": "^4.0.0",
-        "illuminate/support": "~5.1"
+        "illuminate/support": "5.8.*|6.0.*"
```

Comment:
Now that Laravel has moved to semantic versioning, wouldn't `~6.0` be better? Then it will work with new 6.x releases also. And maybe keep `~5.1` as well? No need to cut support for them?
Do you think `~5.1|~6.0` would be sufficient?